server/service/src/test_helpers.rs

```diff
@@ -182,3 +196,85 @@ pub(crate) fn make_movements(stock_line: StockLineRow, date_quantity: Vec<(i64,
         ..Default::default()
     }
 }
+
+pub fn invoice_generate_logs(invoice: &InvoiceRow) -> Vec<ActivityLogRow> {
+    let mut logs = Vec::new();
+
+    let created_log = ActivityLogRow {
+        id: format!("{}_created", invoice.id),
+        r#type: ActivityLogType::InvoiceCreated,
+        user_id: Some("user_account_a".to_string()),
+        store_id: Some(invoice.store_id.clone()),
+        record_id: Some(invoice.id.clone()),
+        datetime: invoice.created_datetime.clone(),
+        changed_to: None,
```

Comment:
Should we use ..Default instead of 'None' here ? 

I've been thinking that all test should use ..Default (to reduce number of places to change when new fields is added)